tsdb/head_append.go

```diff
@@ -514,6 +514,13 @@ func checkHistogramBuckets(buckets []int64) (uint64, error) {
 		return 0, nil
 	}
 
+	checkBucketsErr := func(n int, c int64) error {
```

Comment:
Inline as in, do
```go
return 0, errors.Wrap(
	storage.ErrHistogramNegativeBucketCount,
	fmt.Sprintf("bucket number %d has observation count of %d", i+1, c),
)
```
since it is only used in 1 place

Reply:
Ah yes 🤦🏻 

In a previous iteration it was used in two places but I forgot I removed the first occurance